src/locales/en.json

```diff
@@ -31,16 +31,16 @@
       "after_created": "Once your Note is created, you can find it in your "
     },
     "SavingIndicator": {
-      "saved_just_now": "Saved just now.",
-      "saved_secs_ago": "Saved %{time} sec ago",
-      "saved_min_ago": "Saved %{time} min ago",
-      "saved_mins_ago": "Saved %{time} min ago",
-      "saved_hour_ago": "Saved %{time} hour ago",
-      "saved_hours_ago": "Saved %{time} hours ago",
-      "saved_day_ago": "Saved %{time} day ago",
-      "saved_days_ago": "Saved %{time} days ago",
+      "saved_just_now": "Last update now",
+      "saved_secs_ago": "Last update %{time} sec. ago",
+      "saved_min_ago": "Last update %{time} min. ago",
+      "saved_mins_ago": "Last update %{time} min. ago",
+      "saved_hour_ago": "Last update %{time} hour ago",
+      "saved_hours_ago": "Last update %{time} hours ago",
+      "saved_day_ago": "Last update %{time} day ago",
+      "saved_days_ago": "Last update %{time} days ago",
       "saving": "Saving...",
-      "saving_impossible": "Impossible to save.",
+      "saving_impossible": "Saving... ...",
```

Comment:
```suggestion
      "saving_impossible": "Saving...",
```

Reply:
nope, @joel-costa requires explicit 2 times 3 dots

Reply:
Mmh ok, but that is super strange, and IMO not clear at all. At best it makes it look like we're trying to save again which is not what we do AFAIK?

I'm not sure if it's intentional or not, but in the trello card there is no space between the sets of dots, it more 6 dots than 2 sets of 3 dots.